build/org.eclipse.pde.build.tests/META-INF/p2.inf

```diff
@@ -0,0 +1,2 @@
+requires.0.namespace = org.eclipse.equinox.p2.iu
```

Comment:
This probably is no longer required...

Reply:
Can you then test if it still works without?

Reply:
I have now created
- https://github.com/eclipse-pde/eclipse.pde/pull/1015

as it should actually be fixed at the root and not inside a test.

Reply:
Yes that's right.
IIRC some other tests in PDE and the platform have a similar requirement defined in their pom.
If their tests are changed to be  p2-provisioned it should now not be necessary anymore.

Reply:
Actually it should work regardless of the install mode...